CMakeLists.txt

```diff
@@ -18,8 +18,8 @@ set(CMAKE_AUTORCC ON)
 include(GNUInstallDirs)
 
 option(BUILD_TESTS "Build tests." ON)
-option(BUILD_DOCUMENTATION "Build API documentation." OFF)
-option(BUILD_EXAMPLES "Build API documentation." OFF)
+option(BUILD_DOCUMENTATION "Build API documentation." ON)
+option(BUILD_EXAMPLES "Build API documentation." ON)
```

Comment:
I was actually OK with not building the documentation by default, as this drags an extra dependency on Doxygen, which may not be trivial to satisfy on non-unix platforms (especially getting in on the PATH..)